Dockerfile.local

```
@@ -1,4 +1,4 @@
-FROM golang:1.24.4-bullseye AS base
+FROM golang:1.24.7-bookworm AS base
```

```suggestion
FROM golang:1.24 AS base
```

By only specifying the minor version, it makes the image more reusable for local docker stacks.

---

That makes sense - done 👍